py/lexer.h

```diff
@@ -172,6 +177,9 @@ typedef struct _mp_lexer_t {
     size_t tok_column;          // token source column
     mp_token_kind_t tok_kind;   // token kind
     vstr_t vstr;                // token data
+    vstr_t vstr_postfix;        // postfix to apply to string
```

Comment:
Probably should guard these additions in `#if MICROPY_PY_FSTRING` because otherwise they'll unnecessarily use up RAM.  Then also need to guard all code that uses these new members (hopefully that's not too messy...)